modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `boolean_attributes`                  | Schema option |
+
+| `Class`                               | API           |
+
+| `clearInterval`                       | API           | From `Delay`.
+
+| `clearTimeout`                        | API           | From `Delay`.
+
+| `Color`                               | API           |
+
+| `content_editable_state`              | Option        |
+
+| `debounce`                            | API           | From `Delay`.
+
+| `dependencies`                        | API           | From `AddOnManager`.
+
+| `DomQuery`                            | API           |
+
+| `editor_deselector`                   | Option        | From `EditorManager`.
+
+| `editors`                             | Property      | From `EditorManager`.
+
+| `editor_selector`                     | Option        | From `EditorManager`.
+
+| `elements`                            | Option        | From `EditorManager`.
+
+| `execCallback`                        | API           |
+
+| `file_browser_callback_types`         | Option        |
+
+| `filepicker_validator_handler`        | Option        | Superseded by `file_picker_validator_handler`.
+
+| `force_hex_style_colors`              | Option        |
+
+| `force_p_newlines`                    | Option        | Superseded by `forced_root_block`.
+
+| `forced_root_black: false`            | Option        | `forced_root_black` must be a non-empty string and cannot take a value of `false`.
+
+| `gecko_spellcheck`                    | Option        | Replaced by `browser_spellcheck`.
+
+| `images_dataimg_filter`               | Option        |
+
+| `JSON`                                | API           |
+
+| `JSONP`                               | API           |
+
+| `JSONRequest`                         | API           |
+
+| `mode`                                | Option        | From `EditorManager`.
+
+| `move_caret_before_on_enter_elements` | Schema option |
+
+| `non_empty_elements`                  | Schema option |
+
+| `padd_empty_with_br`                  | Option        |
+
+| `requestAnimationFrame`               | API           | From `Delay`.
+
+| `self_closing_elements`               | Schema option |
+
+| `setIconStroke`                       | API           |
+
+| `setInterval`                         | API           | From `Delay`.
+
+| `setMode`                             | API           |
+
+| `setTimeout`                          | API           | From `Delay`.
+
+| `short_ended_elements`                | Schema option |
+
+| `Sizzle`                              | API           |
+
+| `special`                             | Schema option |
+
+| `text_block_elements`                 | Schema option |
+
+| `text_inline_elements`                | Schema option |
+
+| `throttle`                            | API           | From `Delay`.
+
+| `toolbar_drawer`                      | Option        | Superseded by `toolbar_mode`.
```

Comment:
Should this and other superseded options be linked to the relevant docs?

Reply:
Probably. Could we come back to it later?

Reply:
Deferring